specification.md

```diff
@@ -3421,6 +3446,7 @@ interface FunctionBreakpoint {
 This enumeration defines all possible access types for data breakpoints.
 
 ```typescript
+This enumeration defines all possible access types for data breakpoints.
```

Comment:
this doesn't seem like valid typescript. Shouldn't this be a comment?

It seems to duplicate the line above...

Reply:
Ah, thanks. Generator doesn't behave the way I was expecting.
I'll fix this up.

Reply:
Reverted those...though new version of generator isn't picking up the enum values so still investigating that.